tshirts.cpp

```diff
@@ -20,3 +20,12 @@ void testTshirtSize() {
     assert(size(43) == 'L');
     std::cout << "All is well (maybe!)\n";
 }
+
+void testTshirtSizeEdgeCases() {
+    std::cout << "\nTshirt size edge cases test\n";
+    // Boundary values
+    assert(size(38) == 'M'); // Edge case: exactly 38

```

Comment:
What is the reason to choose a shoulder size of 38 cms as `M`? Can it be `S` as well? Which one is correct?